docs/docs/configuration/configuration-sources.md

```
@@ -415,3 +432,18 @@ export class MyService {
+
+These config sources are not implemented yet, but we plan to implement them in the future:
+
+- Consul / etcd
+- SSM Parameter Store / Vault / GCP/Azure équivalents
+- Kubernetes ConfigMap/Secret
+- HTTP/REST
+- Git (GitOps)
+
+::: warning Wants more?
+Ask us for a custom config source! We can create a custom config source for you, tailored to your needs.
+Contact us at contact-tsed@gmail.com or via [Slack](https://slack.tsed.io/).
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)